python/box.tests/test/org/jetbrains/kotlin/python/test/BasicIrBoxTest.kt

```diff
@@ -42,129 +40,99 @@ import java.io.Closeable
 import java.io.File
 import java.util.regex.Pattern
 
```

Comment:
Out of curiosity: why was it necessary to change these to properties?

Reply:
Not necessary, but it's just a small refactoring here. The advantages of this change are the follows:
1. A specified type allows to catch null values here.
2. Getting rid of backing field (adding `get()` everywhere) is to make a single source of truth: system properties.